registrations/management/commands/repopulate_subscriptions.py

```diff
@@ -50,7 +55,15 @@ def handle(self, *args, **kwargs):
                     'environment variable or --sbm-token is set.')
             client = StageBasedMessagingApiClient(sbm_token, sbm_url)
 
-        registrations = Registration.objects.filter(validated=True)
+        filters = {"validated": True}
+        if query:
+            try:
+                query_key, query_value = query.split("=")
+            except ValueError:
+                raise CommandError(
+                    "Please use the format 'key'='value' for --reg-query")
```

Comment:
Do we perhaps want to use `:` over `=` to differentiate between command line arguments and filtering arguments.